design/crud-api/cmd/server/service.go

```diff
@@ -334,15 +345,23 @@ func main() {
 	}
 	defer neo4jRepo.Close(ctx)
 
+	// Create PostgreSQL client
+	postgresClient, err := postgres.NewClient(*postgresConfig)
+	if err != nil {
+		log.Fatalf("[service.main] Failed to create PostgreSQL client: %v", err)
+	}
+	defer postgresClient.Close()
+
```

Comment:
@Isuru-rangana minor change, but important. We should stick to the convention we have used for the mongo and neo4j. 

Looking at the code, see we use `neo4jRepo` or `mongoRepo` where you have used `postgresClient`. Let's use `postgresRepo`. Also please rename the class`NewClient` method to `NewPostgresRepository`.